ui/src/i18n/ru.json

```
@@ -18,6 +18,10 @@
     "accessories.control.label_on": "Вкл",
     "accessories.control.label_open": "Открыто",
     "accessories.control.label_opening": "Открывается",
+    "accessories.control.label_batteryservice_notcharging": "Не заряжается",
+    "accessories.control.label_batteryservice_charging": "Зарядка",
+    "accessories.control.label_batteryservice_notchargeable": "Не оплачивается",
```

The correct translation in case when charging is not supported: 

```suggestion
"accessories.control.label_batteryservice_notchargeable": "Не поддерживается",
```